docs/variants/novacustom_v560tnx/hardware-matrix.md

```diff
@@ -0,0 +1,137 @@
+
+![](/images/novacustom_v560tnx_rear_view.png)
+
+| No.  | Description                                      |
+|------|--------------------------------------------------|
+| 1.   | RJ-45 LAN Jack                                   |
+| 2.   | HDMI-Out Port                                    |
+| 3.   | Vent/Fan Intake/Outlet                           |
+
+### Left side view
+
+![](/images/novacustom_v560tnx_left_view.png)
+
+| No.  | Description                                      |
+|------|--------------------------------------------------|
+| 1.   | Security Lock Slot                               |
+| 2.   | Vent                                             |
+| 3.   | MicroSD Card Reader                              |
+| 4.   | USB 3.2 Gen 1 Type A Port                        |
+| 5.   | 2-In-1 Audio Jack (Headphone and Microphone)     |
+| 6.   | Speaker                                          |
+
+### Right side view
+
+![](/images/novacustom_v560tnx_right_view.png)
+
+| No.  | Description                                                  |
+|------|--------------------------------------------------------------|
+| 1.   | Speaker                                                      |
+| 2.   | Display Port 1.4 over USB 3.2 Gen 2 Type-C Port              |
+| 3.   | Thunderbolt  4 Combo Port with Power Delivery DC-In          |
+| 4.   | Powered USB 3.2 Gen 2 Type-A Port                            |
+| 5.   | DC-In Jack                                                   |
+
+The graphics used are from pages 16-19 of the official end user manual for the
+V540TNx platforms.
+
+## V560TNC(-S)
+
+| Component                      | Description                                      |
+|--------------------------------|--------------------------------------------------|
+| **CPU**                        | Intel(R) Core(TM) Ultra 7 155H                   |
+|                                | Internal Cooling                                 |
+| **GPU**                        | 1. Intel ARC Graphics Integrated GPU
+|                            | 2. NVIDIA GeForce RTX 4050 discrete GPU|
+| **RAM**                        | Slot 1: W-NM56S516G                              |
+| **SSD**                        | Goodram PX700 1TB                                |
+| **Flash memory**               | GigaDevice 25LB256FYIG                           |
+| **USB pendrives**              | SanDisk Ultra USB 3.0 32 GB                      |
+| **USB Keyboard**               | Logitech, Inc. Keyboard K120                     |
+| **Wireless card 1**            | Intel Wi-Fi 6E AX211                             |
+| **Wireless card 2**            | Intel Wi-Fi 7 BE200                              |
+| **Display**                    | Display 1: HDMI 1920x1080p                       |
+| **Network**                    | Local network wired connection                   |
+| **Internal devices**           | 1. 2880x1800 14 inch screen                      |
```

Comment:
Is that correct? V56 is the 16 inch model ;) Please check the internal panel resolution (e.g. with xrandr)

Reply:
Sorry, copy pasta ;-;

Reply:
[8a5dcc2](https://github.com/Dasharo/docs/pull/870/commits/8a5dcc222026e52e24aec6efd26824e5f82653ae)